app/views/admin/configs/_apple_auth_provider_settings.html.erb

```diff
@@ -0,0 +1,77 @@
+<!--
+Apple's config is different compared to the common OAuth provider implemented
+before, this makes the generalized `auth_provider_settings` not work well so it
+requires a custom partial
+-->
```

Comment:
The gist of why we need this partial is because all OAuth providers use `_key` and `_secret` for easier rendering of the original `_auth_provider_settings.html.erb` partial.

Apple uses different keys (`apple_team_id `, `apple_pem`, `apple_key_id ` & `apple_client_id `) therefore needs a different partial

Reply:
damn you Apple 🤣